SPECS/numpy/generate_source_tarball.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+set -e
+
+SRC_TARBALL="numpy-1.23.4.tar.gz"
+OUT_FOLDER="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+PKG_VERSION="1.23.4"
```

Comment:
I'd remove the hard-coded version, so the `-z` checks below make sense. This will make it clearer to the caller how to use this script and we limit the risk of setting inconsistent arguments (for instance setting the source tarball but forgetting to set the package version).